### ISSUE TITLE
ScrollView and CircleSurfaceEffectBehavior - Scrollbar

### ISSUE THREAD

**Issue by jsobczuk** [Samsung/Tizen.CircularUI#163]
Hi

I am using CircleScrollView connected to bezel on TwoButtonPage:

![scrollbarissue](https://user-images.githubusercontent.com/39958609/53484334-e5150e80-3a83-11e9-989a-887731e0c549.png)

I noticed that this line that connects scrollView to bezel also sets scrollbar to visible.
```cs
cui:CircleSurfaceEffectBehavior.RotaryFocusObject="{x:Reference scrollView}"
```

Could you add property to CircleScrollView that will hide Scrollbar? 'IsScrollbarVisible'?
This is very important for my sample application.

Best regards,
j.sobczuk

**Comment by jkpu** [comment]
Thank you for your issue reporting.

ScrollView.VerticalScrollBarVisibility property can set CircleScrollView's vertical bar visibility.
Please try to set this property in your code. if it doesn't work. please attach your code snippet for investigation.

**Comment by jkpu** [comment]
Patch of VerticalScrollBarVisibility issue was  included version 1.1.0-pre1-00020
